app/src/test/java/com/android/sample/model/recipe/RecipeBuilderTest.kt

```diff
@@ -99,6 +99,28 @@ class RecipeBuilderTest {
     assertEquals("5.00", recipe.price)
   }
 
+  @Test
+  fun `test clear function successfully clears`() {
+    builder.setName("Salad")
+    builder.setInstructions("Mix ingredients")
+    builder.addIngredientAndMeasurement("Lettuce", "100g")
+    builder.setCategory("Vegetarian")
+    builder.setArea("French")
+    builder.setPictureID("http://example.com/image.jpg")
```

Comment:
This test looks good, but it feels a bit fishy, if I may say! Instead of the current approach, would it be more effective to directly assert that each field was correctly reset? While the current setup feels simpler, verifying each field could better showcase the purpose of this unit test and make it clearer what’s being validated. What do you think?

Reply:
Hey! Thanks for the great suggestion! 😊 I went ahead and added assertions for each field to make it clearer what's being validated, as you mentioned. I kept the previous check for the exception too, so we’re covering both angles. Let me know if you think of any other tweaks!